src/screens/rewards/components/RewardsEarnings.tsx

```diff
@@ -61,13 +69,13 @@ export const RewardsEarnings: React.FC<Props> = ({
       totalEarnings.token,
       tokenSymbol
     );
-
-    const formattedTotalEarningsNative = totalEarnings.usd.toLocaleString(
-      'en-US',
-      {
-        style: 'currency',
-        currency: 'USD',
-      }
+    const totalEarningsNative =
```

Comment:
for the non-fallback USD path, you may be looking for `convertAmountAndPriceToNativeDisplay`: https://github.com/rainbow-me/rainbow/blob/develop/src/helpers/utilities.ts#L272